<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QGroupBox" name="groupBox_tableData">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_14"/>
       </item>
       <item>
        <widget class="QTableView" name="tableView"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>462</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="separator"/>
   <addaction name="action_insert"/>
   <addaction name="action_editRec"/>
   <addaction name="action_delete"/>
   <addaction name="separator"/>
   <addaction name="action_scan"/>
   <addaction name="separator"/>
   <addaction name="action_quit"/>
  </widget>
  <action name="action_open">
   <property name="text">
    <string>打开数据库</string>
   </property>
   <property name="toolTip">
    <string>打开数据库</string>
   </property>
  </action>
  <action name="action_editRec">
   <property name="text">
    <string>编缉记录</string>
   </property>
   <property name="toolTip">
    <string>编缉记录</string>
   </property>
  </action>
  <action name="action_insert">
   <property name="text">
    <string>插入记录</string>
   </property>
   <property name="toolTip">
    <string>插入项</string>
   </property>
  </action>
  <action name="action_delete">
   <property name="text">
    <string>删除记录</string>
   </property>
   <property name="toolTip">
    <string>删除项</string>
   </property>
  </action>
  <action name="action_setPhoto">
   <property name="text">
    <string>设置照片</string>
   </property>
   <property name="toolTip">
    <string>清除照片</string>
   </property>
  </action>
  <action name="action_delPhoto">
   <property name="text">
    <string>清除照片</string>
   </property>
   <property name="toolTip">
    <string>清除照片</string>
   </property>
  </action>
  <action name="action_submit">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="toolTip">
    <string>保存修改</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="action_Revert">
   <property name="text">
    <string>取消</string>
   </property>
   <property name="toolTip">
    <string>取消修修改</string>
   </property>
  </action>
  <action name="action_scan">
   <property name="text">
    <string>涨工资</string>
   </property>
   <property name="toolTip">
    <string>涨工资</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>437</x>
     <y>281</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
